### PR TITLE
UNR-3998 Fix 4.25 Android file name error and modify some comments

### PR DESCRIPTION
**Pull request by yujunting**
the file name of Android build with UnrealEngine 4.25-SpatialOSUnrealGDK(GDKShooter-armv7.ap) is different from 4.24-SpatialOSUnrealGDK(GDKShooter-armv7-es2.apk)
change something as comments from old PR
(https://github.com/spatialos/UnrealGDKExampleProject/pull/195)
--------------------
Jira Tickets:
https://improbableio.atlassian.net/browse/UNR-3998

Successful default build:
https://buildkite.com/improbable/unrealgdkexampleproject-nightly/builds/2732

successful completely build:
MAC_BUILD=true
FIREBASE_TEST=true
ENGINE_VERSION="HEAD 4.25-SpatialOSUnrealGDK"
https://buildkite.com/improbable/unrealgdkexampleproject-nightly/builds/2733

### CODE REVIEW

File: ci/nightly.gen.auth.token.yaml [hunk 1]
```diff
@@ -12,7 +12,8 @@ bk_system_error: &bk_system_error
 bk_interrupted_by_signal: &bk_interrupted_by_signal
   exit_status: 15
   limit: 3
-
+  
+# TODO: UNR-3985 Replace the windows agent with a linux agent
```
Comment (jessicafalk) [inline ci/nightly.gen.auth.token.yaml:16]: Add this todo to the other yaml files where we want to replace the windows agent with a linux agent as well

File: ci/setup-and-build.ps1 [hunk 2]
```diff
@@ -188,7 +188,7 @@ pushd "$exampleproject_home"
             "-prereqs", `
             "-nodebuginfo", `
             "-targetplatform=Android", `
-            "-cookflavor=ASTC", `
+            "-cookflavor=ETC2", `
```
Comment (jessicafalk) [inline ci/setup-and-build.ps1:191]: 🎉

Comment (yujunting) [reply ci/setup-and-build.ps1:191]: :grinning:

File: ci/nightly.gen.auth.token.yaml (hunk 1 above)
Comment (yujunting) [reply ci/nightly.gen.auth.token.yaml:16]: ok, all of the windows agents which are planned to switch to Linux agent had been added ticket id before.
please have a look of this PR

File: ci/nightly.template.steps.yaml [hunk 3]
```diff
@@ -65,7 +65,7 @@ steps:
     env:
       ENGINE_COMMIT_HASH: "ENGINE_COMMIT_HASH_PLACEHOLDER"
       # ENGINE_COMMIT_FORMATTED_HASH is a enviroment variable for subsequent steps
-      # It's very similar to ENGINE_COMMIT_HASH, but replace ' ','.','-' as '_' to distinguish engine_version of UnrealEngine.
+      # It's very similar to ENGINE_COMMIT_HASH, but it replace ' ','.','-' with '_' to be able to use it as a buildkite key to distinguish between engine versions.
```
Comment (jessicafalk) [inline ci/nightly.template.steps.yaml:68]:
```suggestion
      # It's very similar to ENGINE_COMMIT_HASH, but it replaces ' ','.','-' with '_' to be able to use it as a buildkite key to distinguish between engine versions.
```

File: ci/run-firebase-test.py [hunk 4]
```diff
@@ -88,8 +88,12 @@ def get_gcs_and_local_path(app_platform, engine_commit_formatted_hash):
     localfilename = ''
     path = 'cooked-%s' % app_platform
     if app_platform == 'android':
-        localfilename = 'GDKShooter-armv7-es2.apk'
-        filename = '%s/Android_ASTC/%s' % (path, localfilename)
+        # UnrealEngine for 4.24-SpatialOSUnrealGDK have different android file name
```
Comment (jessicafalk) [inline ci/run-firebase-test.py:91]:
```suggestion
        # UnrealEngine 4.24 and 4.25 create different android file names
```

**Pull request merged**